added metadata for indexing msdocs code samples

mattsavarino opened the pull request:
Onboarding repo into MS Docs' code samples

mattsavarino committed:
added metadata for indexing msdocs code samples

Dongbumlee reviewed (changes requested):
products:
- azure-blockchain-service
- azure-blockchain-tokens
- azure-cosmos-db

ABT / ABS has been deprecated. it has been built with Azure Cosmos DB, Azure Kubernetes and Azure Container Registry, Azure KeyVault

main language is csharp but included solidity for ERC 721 Smart Contract.

mattsavarino committed:
Metadata edit

Removed deprecated products, and we can follow-up internally w/ Docs team to fix taxonomy for new products/languages.
